shared/src/zeroex_api.rs

```diff
@@ -35,8 +35,6 @@ pub struct SwapQuery {
     pub buy_amount: Option<U256>,
     /// Limit of price slippage you are willing to accept.
     pub slippage_percentage: Slippage,
-    /// Flag to disable checks of the required quantities.
-    pub skip_validation: Option<bool>,
```

Comment:
Removed from the struct because we always set it to false and it doesn't make sense without also having takerAddress. When we want to support takerAddress we can add it back.

Reply:
As a side not, from my quick testing, I'm not sure having `skipValidation=false` and specifying  `takerAddress` as the trader for price estimates will be possible because it requires allowance to be set by the trader on the 0x contract, so it wouldn't work for traders that have never traded on 0x for example.

I agree with the change here.